include/odp/api/spec/schedule.h

```diff
@@ -371,6 +371,31 @@ void odp_schedule_order_lock(uint32_t lock_index);
 void odp_schedule_order_unlock(uint32_t lock_index);
 
 /**
+ * Release existing ordered context lock and acquire a new lock
+ *
+ * This call is valid only when holding an ordered synchronization context.
+ * Release a previously locked ordered context lock and acquire a new ordered
+ * context lock. This call is valid only when there is a single ordered context
+ * lock active. Results are undefined if this call is made when multiple ordered
+ * context locks are acquired in nested fashion.
+ * This call does not release the existing ordered context and only switches
+ * from one ordered lock to another ordered lock within the same ordered
+ * context.
+ *
```

Comment:
It's better to remove this duplicate spec ...
"This call is valid only when there is a single ordered context
lock active. Results are undefined if this call is made when multiple ordered
context locks are acquired in nested fashion.
This call does not release the existing ordered context and only switches
from one ordered lock to another ordered lock within the same ordered
context."

... and refer to lock()/unlock() spec instead: "The operation is equivalent to application calling first odp_schedule_order_unlock(unlock_index) and then odp_schedule_order_lock(lock_index). The same constrains apply to this call than those two."